tests/accelerators/test_accelerator_connector.py

```diff
@@ -332,7 +332,7 @@ def creates_processes_externally(self) -> bool:
             return True
 
     trainer = Trainer(
-        default_root_dir=tmpdir, plugins=[CustomCluster()], fast_dev_run=True, accelerator="ddp_cpu", num_processes=2
+        default_root_dir=tmpdir, plugins=[CustomCluster()], fast_dev_run=True, accelerator="ddp_cpu", devices=2
```

Comment:
```suggestion
        default_root_dir=tmpdir, plugins=[CustomCluster()], fast_dev_run=True, accelerator="cpu", strategy="ddp", devices=2
```
not sure why is it using `ddp_cpu` and a few others. Any specific reason?

Reply:
These tests appear to be explicitly testing `ddp_cpu`

Reply:
exactly